azure-pipelines.yml

```diff
@@ -27,8 +27,7 @@ jobs:
     toxenvs:
       - lint
       - mypy
-      - py36
```

Comment:
Should we keep `py37` in the list? Right now, it's `py36,py36-mindeps`.

It's unlikely that `py37` would fail while `py37-mindeps` and `py310` are both passing, but I can imagine it happening in weird scenarios.

Reply:
It should. My mistake. Let me fix this.